test/development/acceptance-app/dynamic-metadata-error.test.ts

```diff
@@ -0,0 +1,106 @@
+/* eslint-env jest */
+import path from 'path'
+import { sandbox } from 'development-sandbox'
+import { FileRef, nextTestSetup } from 'e2e-utils'
+import { retry } from 'next-test-utils'
+
+describe('dynamic = "error" in devmode', () => {
+  const { next } = nextTestSetup({
+    files: new FileRef(path.join(__dirname, 'fixtures', 'default-template')),
+    skipStart: true,
+  })
+
+  it('should show error overlay when dynamic is forced', async () => {})
```

Comment:
Why is this empty?

Reply:
oops this is unexpected. removed this unrelated one